packages/cyberstorm/src/components/componentStyles/Tag.module.css

```diff
@@ -17,46 +17,49 @@
   --border-color: transparent;
 }
 
-.tag:hover {
+.root:hover {
   --bg-color: var(--color-surface--0);
   --border-color: var(--color-border-3);
 }
 
 /* Different Tag sizes */
 
 .small {
-  gap: var(--size-tiny);
-  padding: var(--size-tiny) var(--text-xxs);
-  font: normal 700 var(--text-xxs) Inter, sans-serif;
-  line-height: var(--size-tiny);
+  gap: var(--gap-tiny);
+  padding: var(--padding-tiny) var(--padding-xxs);
+  font: var(--font-inter);
+  font-weight: var(--font-weight-bold);
+  font-size: var(--text-xxs);
 }
 
 .medium {
```

Comment:
I'm assuming this is the reference one in the Storybook? It looks off from the designs. Here's what I would change with a quick testing in the browser:

- The border radius should be an arbitrary high number to create round ends
- Vertical padding looks more true to the designs as `--padding-xxxxs`
- Gap looks more true as `--gap--xxs`

Reply:
👍